src/Symfony/Bundle/FrameworkBundle/Resources/config/request.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use function Symfony\Component\DependencyInjection\Loader\Configurator\abstract_arg;
```

Comment:
the file should be in the Symfony\Component\DependencyInjection\Loader\Configurator namespace. No need to import the function then.